### DISCUSSION TITLE
AdHosts-Sqlist（For ad-adguardhome）导入软路由adh直接死机了。

### DISCUSSION THREAD

**Discussion by justincnn** [Potterli20/hosts#17] [General]
https://github.com/Potterli20/file/releases/download/ad-hosts-sqlist/ad-adguardhome.txt

把上面连接导入到软路由之后直接死机，断电重启删除才可以正常运行，

这个是因为内容太大了，我的软路由负载不行，还是和现有规则冲突了？

由于死机了，连日志都没法保存查看，求帮助。

目前的软路由为N1,目前同时跑docker下的两个Linux虚拟机都没问题，

**Comment by Potterli20** [comment]
> https://github.com/Potterli20/file/releases/download/ad-hosts-sqlist/ad-adguardhome.txt
> 
> 把上面连接导入到软路由之后直接死机，断电重启删除才可以正常运行，
> 
> 这个是因为内容太大了，我的软路由负载不行，还是和现有规则冲突了？
> 
> 由于死机了，连日志都没法保存查看，求帮助。
> 
> 目前的软路由为N1,目前同时跑docker下的两个Linux虚拟机都没问题，

如果你采用adgh，建议用x86然后内存加到8g内存，因为ad文件去到300m大小，adgh内存就会占用3-4g内存，然后运行加处理起码到5-6g内存。
目前你只用一条规则就够了，你现有的规则我可能有在里面合并。
目前adgh是无法接收大文件处理，这个已经和官方说过，以前的问题官方不处理而已

如果你要用建议用pihole反而我的已经用pihole

**Reply by justincnn** [reply]
8g内存？我家里的群晖也就4G,哈哈

主动放弃了

**Reply by Potterli20** [reply]
哈哈哈，你换pihole不用8g，马上4g，哈哈哈
